real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/simulation/FullSimRunner.java

```diff
@@ -0,0 +1,146 @@
+package com.roboclub.robobuggy.simulation;
+
+import Jama.Matrix;
+import com.roboclub.robobuggy.main.RobobuggyConfigFile;
+import com.roboclub.robobuggy.messages.DriveControlMessage;
+import com.roboclub.robobuggy.messages.EncoderMeasurement;
+import com.roboclub.robobuggy.messages.GpsMeasurement;
+import com.roboclub.robobuggy.messages.SteeringMeasurement;
+import com.roboclub.robobuggy.nodes.baseNodes.BuggyBaseNode;
+import com.roboclub.robobuggy.nodes.baseNodes.PeriodicNode;
+import com.roboclub.robobuggy.nodes.localizers.LocTuple;
+import com.roboclub.robobuggy.nodes.localizers.LocalizerUtil;
+import com.roboclub.robobuggy.nodes.localizers.RobobuggyKFLocalizer;
+import com.roboclub.robobuggy.nodes.localizers.UTMTuple;
+import com.roboclub.robobuggy.nodes.planners.WayPointFollowerPlanner;
+import com.roboclub.robobuggy.nodes.planners.WayPointUtil;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ros.Publisher;
+import com.roboclub.robobuggy.ros.Subscriber;
+
+import java.io.FileNotFoundException;
+import java.util.Date;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Created by vivaanbahl on 4/2/17.
+ */
+public class FullSimRunner extends BuggyBaseNode {
+
+    private static final double WHEELBASE_IN_METERS = 1.13; // meters
```

Comment:
These are the same parameters as the tester. Should these all be in one place somewhere?